src/index.js

```diff
@@ -75,4 +75,4 @@ app.ports.stopSequence.subscribe(function(){
   }
 });
 
-registerServiceWorker();
+// registerServiceWorker();
```

Comment:
It is okay to delete this code if you don't want it. The project is under version control so we can get it back anytime.